pythermalcomfort/charts/heatmap.py

```diff
@@ -0,0 +1,215 @@
+from math import ceil, floor
+import numpy as np
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+from pythermalcomfort.charts.theme import (
+    tight_margins,
+    index_mapping_dictionary,
+)
+
+
+def heatmap(df, var, global_local, si_ip, show_summary=False):
+    """Generates a heatmap with the desired variable.
+
+    Parameters
+    ----------
+    df : DataFrame
+        A DataFrame containing the input data. Must have the following columns:
+        'DateTime' (Date and time), 'Hour' (Hour of the day), 'Month Name' (Month name), 'Day' (Day of the month),
```

Comment:
Please add the each column name on a different row if possible

Reply:
Do we really need all these inputs? Can't we extract the our month name and the day from the date time?